Door height

lolyay270 opened the pull request:
## Overview
Players can no longer smash their heads on the top of the door frames in tutorial

<!--Remove headings that dont apply to your PR-->
### Changed
- Adjusted the gap in the `DoorWall` 3D model
- Adjusted `DoorWall` prefab
  - Have new model
  - Door sized to fit
 - Re-baked `Tutorial` lighting since changing the model broke it
### Fixed
- Tutorial Exit signs lighting sometimes not showing

<!--If there is no way to test, remove this section-->
## How to test changes
1. Pull `08-jb-door-height` branch
2. Open `08-jb-door Tutorial` scene
4. Ensure you are standing 
5. Find any door and try to walk through it with continuous movement
6. Notice you walk through no problem

## Issue number and link
In the tutorial, players cannot walk through the door frames
because the character is colliding with the wall above the door
Closes #92
<!--Please ensure your issue is fully linked after creating, 
GitHub has been failing to link them when we use "Closes"-->

lolyay270 committed:
feat(door): new doorWall model with taller hole

lolyay270 committed:
feat(door): new wall in prefab, adjusted in tutorial

lolyay270 committed:
feat(tutorial): rebaked lighting

AmberSolstice reviewed (approved):
Doors are working great and you can not clip thro the walls. Really good job.

Christian-Irvine reviewed (changes requested):
This is very nice as it was kinda goofy walking into the top of the door. It seems as though there is an issue where you can walk through walls because of a tag not being set on them. It also seems like there is an issue where the wall sticks out in the first main room of the tutorial. Otherwise the features pretty nice. Definitely an improvement.

lolyay270 committed:
fix (doorWall): now has collision

lolyay270 commented:
> This is very nice as it was kinda goofy walking into the top of the door. It seems as though there is an issue where you can walk through walls because of a tag not being set on them. It also seems like there is an issue where the wall sticks out in the first main room of the tutorial. Otherwise the features pretty nice. Definitely an improvement.

### Could I please have a re-review:
I have updated the tag on the wall, to `Geometry`, so it matches the other walls in tutorial
Also the wall's mesh in the mesh collider was somehow missing so I replaced that

Christian-Irvine reviewed (approved):
Awesome work Jenna, thanks for the changes. I'm happy to approve for merging.
